crates/matrix-sdk-base/src/store/send_queue.rs

```diff
@@ -365,6 +369,9 @@ pub struct DependentQueuedRequest {
     /// If the parent request has been sent, the parent's request identifier
     /// returned by the server once the local echo has been sent out.
     pub parent_key: Option<SentRequestKey>,
+
+    /// The time that the request was original attempted.
+    pub enqueue_time: Option<MilliSecondsSinceUnixEpoch>,
```

Comment:
I'm not sure about this, but since this is (de)serialized: should this have a `#[serde(default)]` annotation to indicate that if the field is missing, it should be `None` — or is that the default (heh) behavior for `Option`s?

Reply:
Yeah, it seems like generally we have annotations for serializing (to skip if it is None).  I'll add that here for consistency.